services/py-api/src/database/repository/participants_repository.py

```diff
@@ -48,14 +51,16 @@ async def create(
                 # If the team_id is passed as a kwarg the participant will be inserted in the given team
                 team_id=kwargs.get("team_id"),
             )
-            LOG.debug("Inserting participant...", participant=participant.dump_as_json())
+            LOG.debug("Inserting participant...", participant=participant.dump_as_json(), session_id=session_id)
             await self._collection.insert_one(document=participant.dump_as_mongo_db_document(), session=session)
             return Ok(participant)
         except DuplicateKeyError:
-            LOG.warning("Participant insertion failed due to duplicate email", email=input_data.email)
+            LOG.warning(
+                "Participant insertion failed due to duplicate email", email=input_data.email, session_id=session_id
+            )
             return Err(DuplicateEmailError(input_data.email))
         except Exception as e:
-            LOG.exception("Participant insertion failed due to err {}".format(e))
+            LOG.exception(f"Participant insertion failed due to err {e}", session_id=session_id)
```

Comment:
is this working? I think the logging might fail with this f-string. Test it pls

Reply:
Yes it is working! I tested it.